tmol/tests/data/__init__.py

```diff
@@ -62,6 +62,13 @@ def ubq_system():
     return read_pdb(pdb.data["1ubq"])
 
 
+@pytest.fixture(scope="session")
+def disulfide_res():
```

Comment:
Named this 'disulfide' since that probably is a more useful name to anyone in the future looking for a fixture with disulfides in it.